composer.json

```diff
@@ -15,7 +15,7 @@
     ],
     "minimum-stability": "stable",
     "require": {
-        "php": ">=8.0",
```

Comment:
why this change? we are not changing anything related to PHP version

Reply:
Laravel 10 needs PHP 8.1 minimum, so making this change to allow PHP version 8.*